### ISSUE TITLE
Empty .mp3 in german corpus

### ISSUE THREAD

**Issue by zngguvnf** [mozilla/voice-web#2215]
Hi all,

**tl;dr**
there are empty .mp3 files in the german corpus (version: de_340h_2019-06-12), most of the files are named in the validated.tsv file.

List of empty files:

```
./clips/common_voice_de_17876483.mp3
./clips/common_voice_de_17876495.mp3
./clips/common_voice_de_17876535.mp3
./clips/common_voice_de_17876536.mp3
./clips/common_voice_de_17876537.mp3
./clips/common_voice_de_17876538.mp3
./clips/common_voice_de_17876539.mp3
./clips/common_voice_de_17876540.mp3
./clips/common_voice_de_17876541.mp3
./clips/common_voice_de_17876542.mp3
./clips/common_voice_de_17876543.mp3
./clips/common_voice_de_17876544.mp3
./clips/common_voice_de_17876545.mp3
./clips/common_voice_de_17876546.mp3
./clips/common_voice_de_17876547.mp3
./clips/common_voice_de_17876548.mp3
./clips/common_voice_de_17876549.mp3
./clips/common_voice_de_17876550.mp3
./clips/common_voice_de_17876551.mp3
./clips/common_voice_de_17876552.mp3
./clips/common_voice_de_17876553.mp3
./clips/common_voice_de_17876554.mp3
./clips/common_voice_de_17876555.mp3
./clips/common_voice_de_17876556.mp3
./clips/common_voice_de_17876557.mp3
```

**Steps to reproduce:**

1. Download german common voice corpus from [here](https://voice.mozilla.org/de/datasets) (version: de_340h_2019-06-12).

2. Check md5sum
```
$ md5sum de.tar.gz 
246ebee04a5aa95318b4aae7b5fda562  de.tar.gz
```

3. Extract folder
```
tar xzvf de.tar.gz 
```

4. Calculate file size
e.g.

```
find . -name '*.mp3' | xargs du -sh | sort -h | vim -
```

You will see size 0 for the files, named above.

5. Where are the empty files called
```
$grep -nrw ./*.tsv -e 'common_voice_de_17876483.mp3
./validated.tsv:88256:875b81164215d345c57b0e04243b6d093db729dd43a07061e807dd04f622215a332c1af9989e9b3db5ac7d54aba2286274204b5a729ad354746e51ca0c121fa6	common_voice_de_17876483.mp3	Die Kobaltmine ist vor Jahrzehnten aufgegeben worden.20	twenties	male	germany
```

Most of the files are named in the validated.tsv file. So most likely this bug will crash your training.

Because just 25/293932 files are affected, maybe deleting them is an option.

If you agree with my guess and point me to the right place I would be happy to create a pull request.

**Comment by moz-bozden** [comment]
Bad audio have already been removed in further releases.
Closing.
